Mathlib/Geometry/Manifold/Instances/Sphere.lean

```diff
@@ -403,15 +404,15 @@ instance EuclideanSpace.instSmoothManifoldWithCornersSphere {n : ℕ} [Fact (fin
       rfl)
 
 instance (n : ℕ) :
-    SmoothManifoldWithCorners (𝓡 n) (sphere (0 :  EuclideanSpace ℝ (Fin (n + 1))) 1) :=
+    IsManifold (𝓡 n) ω (sphere (0 :  EuclideanSpace ℝ (Fin (n + 1))) 1) :=
   haveI := Fact.mk (@finrank_euclideanSpace_fin ℝ _ (n + 1))
-  EuclideanSpace.instSmoothManifoldWithCornersSphere
+  EuclideanSpace.instIsManifoldSphere
 
 /-- The inclusion map (i.e., `coe`) from the sphere in `E` to `E` is smooth. -/
```

Comment:
You could even say "is analytic". (Not sure if that's clearer.) Or mention both.

Reply:
If so, also please update the module doc-string to this file.